Examples.fshx

```diff
@@ -53,32 +133,154 @@ Instance: mCODEKarnofskyPerformanceStatusExample01
 InstanceOf: KarnofskyPerformanceStatus
 * meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/KarnofskyPerformanceStatus"
 * status = #final "final"
+* category = #survey "survey"
 * method = SCT#5880005 "Clinical examination"
+* code = LNC#89245-5 "Karnofsky Performance Status [Interpretation]"
 * subject = Reference(mCODEPatientExample01)
 * effectiveDateTime = "2019-04-01"
 * performer = Reference(mCODEPractitionerExample01)
 * valueCodeableConcept = LNC#LA29177-5 "Normal activity with effort; some signs or symptoms of disease"
 
+
 Instance: mCODECancerRelatedMedicationStatementExample01
 InstanceOf: CancerRelatedMedicationStatement
 * meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/CancerRelatedMedicationStatement"
 * status = #active "active"
-* medicationCodeableConcept = RXN#93698 "Carboplatin Injectable Solution [Paraplatin]"
+* category = #community "community"
+* medicationCodeableConcept = RXN#349472 "gefitinib 250 MG Oral Tablet"
 * subject = Reference(mCODEPatientExample01)
 * effectiveDateTime = "2019-04-01"
 * dateAsserted = "2019-04-01"
-* extension.valueCodeableConcept = SCT#373808002 "Curative - procedure intent"
 * extension[TreatmentIntent].valueCodeableConcept = SCT#373808002 "Curative - procedure intent"
+* dosage.text = "250mg orally once daily with or without food"
+* dosage.route = SCT#26643006 "Oral use"
+* dosage.dose[x].system = UCUM#mg "mg"
+* dosage.dose[x].value = "250"
 
 
 Instance: mCODECancerRelatedSurgicalProcedureExample01
 InstanceOf: CancerRelatedSurgicalProcedure
 * meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/CancerRelatedSurgicalProcedure"
 * status = #completed "completed"
+* category = SCT#387713003 "Surgical procedure"
```

Comment:
Should this be in the profile?  Is it already there?

Reply:
It is not in the profile. I can add it?

Reply:
Yes, for both surgical procedure and radiation procedure, the category should be in the profile. Let's merge this, then make a change.